set-of-emails/invalid_bounce.txt

```diff
@@ -0,0 +1 @@
+Date: Tue, 06 Feb 2024 07:23:29 +0000
From: Mail Delivery System <example@mailer.com>
To: example@mailer.com
Message-ID: <6667755343456.9B6BD4004BA9@relmlir5.idc.mailer.com>
Subject: Undelivered Mail Returned to Sender
Mime-Version: 1.0
Content-Type: multipart/mixed;
 boundary="--==_mimepart_65c1fd2aee126_72f949768";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
X-OUTREACH-SENT: true


----==_mimepart_65c1fd2aee126_72f949768
Content-Type: multipart/alternative;
 boundary="--==_mimepart_65c1fd2aee07b_72f9497553";
 charset=UTF-8
Content-Transfer-Encoding: 7bit


----==_mimepart_65c1fd2aee07b_72f9497553
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

This is the mail system at host relmlir5.idc.mailer.com.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<sp.tan@mailer.com>: unknown user: "sp.tan"

----==_mimepart_65c1fd2aee07b_72f9497553
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit


----==_mimepart_65c1fd2aee07b_72f9497553--

----==_mimepart_65c1fd2aee126_72f949768--
```

Comment:
This file will be moved/renamed to `set-of-emails/maildir/<somewhere>` after merging code